demo/admin/src/products/future/ProductsGrid.cometGen.ts

```diff
@@ -14,4 +14,8 @@ export const ProductsGrid: GridConfig<GQLProduct> = {
         { type: "date", name: "availableSince", width: 140 },
         { type: "dateTime", name: "createdAt", width: 170 },
     ],
+    actions: {
+        columnWidth: 116,
```

Comment:
Why is this needed?

Reply:
DataGrid columns do not scale automatically, depending on their content. 
We need to set the width manually depending on the number of `IconButton` components shown to prevent them from being cut off.

Reply:
how does mui `type: "actions"` solve this width issue? Do you also have to specify it manually?

Reply:
It doesn't, you also have to specify it manually.